### PR TITLE
Add ability to set font size in plot

### PR DESCRIPTION
**Pull request by this-josh**
This PR permits you to set the leaflet font size. This is helpful to make plots more legible in certain circumstances.

### CODE REVIEW

File: folium/folium.py [hunk 1]
```diff
@@ -235,6 +236,7 @@ def __init__(
         height: Union[str, float] = "100%",
         left: Union[str, float] = "0%",
         top: Union[str, float] = "0%",
+        font_size: str = "1rem",
```
Comment (martinfleis) [inline folium/folium.py:239]: This needs to be on the bottom as the last argument so it does not break potential use of positional args.

It also needs to be added to the docstring above.

**Review by martinfleis — COMMENTED** [review]
Nice change! Minor comment in the code.

File: folium/folium.py (hunk 1 above)
Comment (this-josh) [reply folium/folium.py]: Good catch, I've implemented these changes

**Review by martinfleis — APPROVED** [review]

**Review by Conengmo — COMMENTED** [review]
Thanks for your effort @this-josh! It’s a nice improvement and I’m sure we can merge it soon.

 I found a typo, and I added a question, hope to hear your thinking on it.

File: folium/utilities.py [hunk 2]
```diff
@@ -425,3 +425,13 @@ class JsCode:
...
     def __init__(self, js_code: str):
         self.js_code = js_code
+
+
+def parse_font_size(value: Union[str, int, float]) -> str:
+    """Parse a font size value."""
+    if isinstance(value, (int, float)):
+        return f"{value}rem"
+
+    if "rem" not in value:
```
[the remark this answers is not in the document]
Comment (this-josh) [reply folium/utilities.py]: Good point, I've updated it to accept `em` and `px`.

[the remark this answers is not in the document]
Comment (Conengmo) [reply folium/utilities.py]: Thanks! The default could still be rem if you like.
I noticed some test may fail, once that’s solved I’ll go ahead and merge it!

File: tests/test_utilities.py [hunk 3]
```diff
@@ -218,7 +219,7 @@ def test_escape_double_quotes(text, result):
 def test_javascript_identifier_path_to_array_notation(text, result):
     assert javascript_identifier_path_to_array_notation(text) == result
 
-
+
```
Comment (Conengmo) [inline tests/test_utilities.py:222]:
```suggestion

```

File: tests/test_utilities.py [hunk 4]
```diff
@@ -218,7 +219,6 @@ def test_escape_double_quotes(text, result):
 def test_javascript_identifier_path_to_array_notation(text, result):
     assert javascript_identifier_path_to_array_notation(text) == result
 
-
 def test_js_code_init_str():
```
Comment (Conengmo) [inline tests/test_utilities.py:222]:
```suggestion

def test_js_code_init_str():
```

**Comment by Conengmo** [comment]
Thanks @this-josh!

**Pull request merged**